content/08-cardano-sidechains/03-example-evm-sidechain/07-api.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: JSON-RPC API
+metaTitle: JSON-RPC API
+---
+
+Interacting with the sidechain requires sending requests to specific JSON-RPC API methods. The EVM sidechain testnet supports most standard Ethereum JSON-RPC API endpoints with a few exceptions described below.
+
+## Specification
+
+The EVM sidechain testnet supports [OpenRPC](https://open-rpc.org/)'s `rpc.discover` service discovery method that will return the OpenRPC scheme for the JSON-RPC API. You can easily validate and preview the schema on the [OpenRPC Playground](https://playground.open-rpc.org/?schemaUrl=https://faucet.sidechain.evmtestnet.iohkdev.io/).
+
+## Supported methods
+
+The list below shows which Ethereum API methods are currently implemented in the Cardano EVM sidechain. For details of each method, you can refer to the [JSON-RPC specification](https://playground.open-rpc.org/?schemaUrl=https://faucet.sidechain.evmtestnet.iohkdev.io/).
+
+*If your product requires a method that is not currently implemented, please contact Input Output Global support.*
+
+# Available methods
+
+- `eth_getBlockByHash`
+- `eth_getBlockByNumber`  
+- `eth_sendTransaction`  
+- `eth_estimateGas`  
+- `eth_getCode`  
+- `eth_sendRawTransaction`  
+- `eth_getTransactionReceipt`  
+- `eth_call`  
+- `eth_getBalance`  
+- `eth_getStorageAt`  
+- `eth_getBlockTransactionCountByHash`  
+- `eth_getBlockTransactionCountByNumber`
+- `eth_accounts`  
+- `eth_blockNumber` 
+- `eth_sign`
+- `eth_getTransactionByHash`  
+- `eth_getTransactionCount`  
+- `eth_chainId`  
+- `net_version`  
+- `web3_clientVersion`
+- `txpool_content`
+
+## Bridge methods
+
+These methods are specifically implemented to support bridge operations.
+
+- `sidechain_getCandidates`
+- `sidechain_getCommittee`
+- `sidechain_getMainchainStatus`
+
```

Comment:
!!!!!!! Missing a lot of methods, that can be found in https://playground.open-rpc.org/?schemaUrl=https://faucet.sidechain.evmtestnet.iohkdev.io/
!!!!!!!
I.e. getCurrentCandidates, getEpochSignatures, getOutgoingTransactions, merkle proof, params, pendingTxs, signatures to Upload and getStatus.

Reply:
Based on David's suggestion we'll just delete maintaining a list of the methods and just use OpenRPC for more efficient maintenance.